data/sql/updates/pending_db_world/fixtext7998v2.sql

```diff
@@ -0,0 +1 @@
+UPDATE `creature_text` SET `Text` = 'FIRE IN THE HOLE!', `Type` = 14, `BroadcastTextId` = 32326 WHERE (`CreatureID` = 7998) AND (`GroupID` = 19) AND (`ID` = 0);
```

Comment:
It's good, just adding this here because I noticed it
```suggestion
UPDATE `creature_text` SET `Text` = 'FIRE IN THE HOLE!', `Type` = 14, `BroadcastTextId` = 32326 WHERE (`CreatureID` = 7998) AND (`GroupID` = 19) AND (`ID` = 0);
UPDATE `creature_text` SET `BroadcastTextId` = 4446 WHERE `CreatureID` = 7998 AND `GroupID` = 20 AND `ID` = 0;
```